R/build-home.R

```diff
@@ -3,185 +3,8 @@
 #' This function generates the home page, converts `.md` files found in the
 #' package root (and in `.github/`), and builds an authors page from
 #' `DESCRIPTION` and `inst/CITATION` (if present).
-#'
```

Comment:
I find it very hard to write so long text in `.R` + it means the components could be re-used.